Makefile.Inc

```diff
@@ -0,0 +1,59 @@
+PROJ_DIR := $(dir $(realpath $(firstword $(MAKEFILE_LIST))))
+BIN_DIR ?= $(PROJ_DIR)build/
+
+export CGO_LDFLAGS := -L$(BIN_DIR) -Wl,-rpath,\$$ORIGIN $(CGO_LDFLAGS_EXT)
+export GOOS
+export GOARCH
+export GOARM
+export BIN_DIR
+
+ifeq ($(OS),Windows_NT)
+  HOST_OS := windows
+else
+  HOST_OS := $(shell uname | tr [A-Z] [a-z])
+endif
+
+ifeq ($(GOOS),)
+  GOOS := $(HOST_OS)
+endif
+
+ifeq ($(GOOS),windows)
+  platform := windows
+  export PATH := $(PATH):$(PROJ_DIR)build
+  EXE := .exe
+else
+  TEMP := /tmp
+  ifeq ($(GOOS),darwin)
+	  platform := macos
+  else
+      platform := linux
+  endif
+endif
+
+$(info "OS: $(OS), HOST_OS: $(HOST_OS), GOOS: $(GOOS), GOARCH: $(GOARH), BIN_DIR: $(BIN_DIR), platform: $(platform)")
+
+GPU_SETUP_REV = 0.1.20
+GPU_SETUP_ZIP = libgpu-setup-$(platform)-$(GPU_SETUP_REV).zip
+GPU_SETUP_URL_ZIP = https://github.com/spacemeshos/gpu-post/releases/download/v$(GPU_SETUP_REV)/$(GPU_SETUP_ZIP)
+ifeq ($(platform), windows)
+  GPU_SETUP_LIBS = gpu-setup.dll

```

Comment:
Aren't we missing gpu-setup.lib here?

Reply:
Because gpu-setup.lib is build by VS and in MSCOFF format, but go uses GNU toolchain and requires DWARF. Icreated required library and you can add to zip. It's here https://github.com/sudachen/post/blob/playground/gpu/libgpu-setup-win64.a

Reply:
No. It's not usable by go because it uses gnu toolchain, but gpu-setup.lib is built by VS and in the MSCOFF format.

Reply:
Good to know - thanks for the info.